source/how-tos/update-firmware.rst

```diff
@@ -23,32 +23,43 @@ Creating firmware update capsule image requires the following steps:
 
 #. Run capsule tool to generate capsule image
 
-  Capsule tool (``GenCapsuleFirmware.py``) creates a capsule image that can be processed by |SPN| in firmware update flow::
+  Capsule tool (``GenCapsuleFirmware.py``) creates a capsule image that can be processed by |SPN| in firmware update flow.
 
-    usage: GenCapsuleFirmware.py [-h] -b IMAGE [-t {b,c}] -k PRIVKEY -o NEWIMAGE
-                                 [-q]
+  Capsule tool is capable of incorporating multiple firmware images into single capsule binary. Each firmware is identified and included in the capsule image using a GUID.
+
+    usage: GenCapsuleFirmware.py [-h] -p BIOS <BIOS_IMAGE> -p <GUID> <FW IMAGE BINARY 1>.....-p <GUID> <FW IMAGE BINARY n> -k PRIVKEY -o NEWIMAGE [-q]
 
     optional arguments:
       -h, --help            show this help message and exit
-      -b IMAGE, --image IMAGE
-                            Raw image into firmware update capsule image
-      -t {b,c}, --type {b,c}
-                            Image type b:Bootloader c:CfgData
+      -p  <GUID> <Payload Image>, 
+                            Payload image that goes into firmware update capsule
       -k PRIVKEY, --priv_key PRIVKEY
                             Private RSA 2048 key in PEM format to sign image
       -o NEWIMAGE, --output NEWIMAGE
                             Output file for signed image
       -q, --quiet           without output messages or temp files
 
+  For example, the following command generates a capsule image (``FwuImage.bin``) containing an IFWI image (``sbl.bios.bin``), CSME image (``csme.bin``), CSME Driver (``csme_driver.bin``) and firmware image (``fwimage.bin``) signed by key ``TestSigningPrivateKey.pem``::

```

Comment:
It should be "CSME firmware update driver".

Reply:
Done